microsite/core/Footer.js

```diff
@@ -54,6 +56,13 @@ class Footer extends React.Component {
             <a href={this.props.config.fossWebsite}>
               Open Source @ {this.props.config.organizationName}
             </a>
+      
+      <a href="https://engineering.atspotify.com/">Spotify Engineering Blog</a>
+      </a>
+
+      <a href="https://developer.spotify.com/">Spotify for Developers</a>
+</a>
+      
```

Comment:
There are two extra `</a>` that need to be removed.

You may also need to align these with the preceding code by adding more spaces so they line up.

Reply:
Will do, thank you.

Reply:
Not sure we can actually put that here? The microsite is supposed to be vendor neutral

Reply:
See https://github.com/cncf/foundation/blob/master/website-guidelines.md